packages/regex/src/__tests__/index.ts

```diff
@@ -41,6 +41,7 @@ import {
   sixDigitsCode,
   spaces,
   uppercaseBasicDomain,
+  uppercaseBasicSubdomain,
```

Comment:
What is a subdomain ? Only xxx.foo.tld ? Or can be nested like (zzz*n).yyy.xxx.foo.tld ?

Reply:
Yes it can be nested.